phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/MainController.java

```diff
@@ -64,26 +77,28 @@ public MainController(String applicationName,
                 themeManager,
                 javaFxSettingsManager);
 
+        this.themeManager = themeManager;
         this.javaFxSettingsManager = javaFxSettingsManager;
 
-        installationsController.setOnInstallationAdded(mainWindow::showInstallations);
-
-        appsController.setOnAppLoaded(() -> {
-            containersController.loadContainers();
+        repositoryManager.addCallbacks(this::setDefaultCategoryIcons, e -> {
```

Comment:
I think we should add an `addCallbacks` method to the `RepositoryManager` class which doesn't require an error callback

Reply:
Can you open a new issue for this?

Reply:
#1624